Framework/DataHandling/src/LoadEMU.cpp

```diff
@@ -231,8 +268,67 @@ class ConvertTOF {
     double t = invert(tsec, tn, t0);
     double tof = m_L0 / v1(t) + m_L2[detID] / m_v2;
 
-    return tof * 1.0e6;
+    return TofData(t * 1.0e6, tof * 1.0e6);
+  }
+};
+
+// calculate mean of a subset of the vector
+double maskedMean(std::vector<double> &vec, std::vector<bool> &mask) {
```

Comment:
Can these both be `const std::vector<double> & `?

Reply:
Yes, changed.